ffs/scheduler/scheduler.go

```diff
@@ -163,6 +175,50 @@ func (s *Scheduler) Cancel(jid ffs.JobID) error {
 	return nil
 }
 
+// GCStaged runs a unpinned garbage collection of stage-pins.
+func (s *Scheduler) GCStaged(ctx context.Context) ([]cid.Cid, error) {
+	return s.gcStaged(ctx, 0)
```

Comment:
Maybe I'm missing something here, but would expect `0` to be `s.gc.StageGracePeriod`

Reply:
Actually, this is on purpose. This API is for the manual GC call that I feel should clean all staged stuff (grace period 0).
I'd imagine that the user would like the manual option to be more direct clean.

Reply:
👍